src/apps/ubeswap/positions.ts

```diff
@@ -57,23 +60,43 @@ async function getPoolPositionDefinition(
   })
   const position: AppTokenPositionDefinition = {
     type: 'app-token-definition',
-    network,
+    networkId,
     address: poolAddress.toLowerCase(),
     tokens: [token0Address, token1Address].map((token) => ({
       address: token.toLowerCase(),
-      network,
+      networkId,
     })),
-    displayProps: ({ resolvedTokens }) => {
-      const token0 = resolvedTokens[token0Address.toLowerCase()]
-      const token1 = resolvedTokens[token1Address.toLowerCase()]
+    displayProps: ({ resolvedTokensByTokenId }) => {
+      const token0 =
+        resolvedTokensByTokenId[
+          getTokenId({
+            networkId,
+            address: token0Address,
+            isNative: isNative({
+              networkId: networkId,
+              address: token0Address,
+            }),
+          })
+        ]
+      const token1 =
+        resolvedTokensByTokenId[
+          getTokenId({
+            networkId,
+            address: token1Address,
+            isNative: isNative({
+              networkId: networkId,
+              address: token1Address,
+            }),
+          })
+        ]
```

Comment:
nit: I wonder if we could find a way to keep this short.

Maybe `isNative` is a bit redundant? 
Or I guess we could have this function be called with a helper function that's more succint? `getResolvedToken(networkId, address)` which would use `resolvedTokensByTokenId` under the hood.

Feel free to ignore here, we can do that in a follow up PR.

Reply:
I tried out making it more succinct by calling isNative from getTokenId